SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestPawnBase_RepGraphAlwaysReplicate.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "GameFramework/DefaultPawn.h"
+#include "ReplicatedTestPawnBase_RepGraphAlwaysReplicate.generated.h"
+
+/**
+ * A replicated Pawn which is always replicated by RepGraph regardless of client view.
+ */
+UCLASS()
+class AReplicatedTestPawnBase_RepGraphAlwaysReplicate : public ADefaultPawn
```

Comment:
Maybe similar to Mike's comment this could be `ATestPawnBase_RepGraphAlwaysReplicate`

Reply:
Sure, just wanted it to have similar name to the actor one